### ISSUE TITLE
`UndefVarError: nil not defined` when sending line/region with two prefix args

### ISSUE THREAD

**Issue by sjkobayashi** [gcv/julia-snail#109]
After setting `julia-snail-popup-display-eval-results` to `nil`, I receive the following Julia error when using `C-u C-u C-c C-l` or `C-u C-u C-c C-r`

```
UndefVarError: nil not defined

top-level scope at none:1
eval at boot.jl:373 [inlined]
eval_in_module(fully_qualified_module_name::Vector{Symbol}, expr::Expr) at JuliaSnail.jl:226
macro expansion at JuliaSnail.jl:875 [inlined]
(::Main.JuliaSnail.var"#29#33"{Sockets.TCPSocket})() at task.jl:429
```

**Comment by gcv** [comment]
Thank you for the bug report, and apologies for the delayed reply. I think I fixed this in ef72f07. This should be available in MELPA in about two hours. Please try the latest version of Snail and see if that change fixes the problem for you.

**Comment by sjkobayashi** [comment]
Thank you. I just confirmed that the issue is fixed!
